src/partials/contacts.html

```diff
@@ -0,0 +1,48 @@
+<section class="contacts">
+  <div class="container">
+    <h2 class="contacts__main-header">Contacts</h2>
+    <ul class="contacts__list">
+      <li class="contacts__list-item">
+        <h3 class="contacts__label">Cafe</h3>
+        <h4 class="contacts__location">Chicago</h4>
+        <p class="contacts__text">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+        <p class="contacts__days">Monday - Friday</p>
```

Comment:
"contacts__days" ma to samo stylowanie co "location, dlatego zamiast <p> proponuję<h4>